sidebarsDocs.js

```diff
@@ -198,6 +200,19 @@ module.exports = {
       collapsed: false,
     },
   ],
+  policies: [
+    {
+       type: 'link',
+       label: '<- Back',
+       href: '/use-the-network/policies'
+     },
+     {
+       type: 'category',
+       label: 'Policies',
+       items: ['use-the-network/policies/policies','use-the-network/policies/privacy', 'use-the-network/policies/terms'],
```

Comment:
We usually don't put the top level item, 'use-the-network/policies/policies', in the sub menu.

Reply:
resolved, thanks